bundles/org.openhab.transform.exec/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -6,5 +6,6 @@
 	<type>transformation</type>
 	<name>Exec transformation</name>
 	<description>Transforms an input string with an external program.</description>
+	<connection>local</connection>
```

Comment:
Or is it `none`? I assume the external program is being executed on the machine running openHAB, not through the network?

Reply:
I don't have a strong opinion here. For me, transforms typically are handled within openHAB -> none. The exec transform is a bit different, interacting with the system outside of openHAB -> so I thought local would be fine.

For the TTS addons we could have the same discussion - but most of the other TTS systems were tagged as local as well, so I have put local as well (although none would have been an option as well for some of them).

The connection attribute is a bit dependent on the view point of the author.